ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/PromotionalOfferScreen.kt

```diff
@@ -85,17 +87,16 @@ internal fun PromotionalOfferScreen(
         ) {
             Text(promotionalOfferData.localizedPricingPhasesDescription)
         }
-        OutlinedButton(
+
+        TextButton(
             onClick = { onDismiss() },
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(bottom = 16.dp),
```

Comment:
Do we need to add some extra padding to the button? Looking at the diff in emerge, seems that it's too close to the bottom IMO?

Reply:
You're right, I'll put it back